src/containers/app/AppContainer.js

```diff
@@ -80,6 +81,9 @@ const AppContainer = () => {
                   <DashboardContainer />
                 </Auth0AdminRoute>
               )} />
+          <Route path={Routes.ENROLLMENT_LINK}>
+            <EnrollmentLink />
+          </Route>
```

Comment:
This will not work if the user doesn't have a OL account (most likely to be the case) / doesn't have access to a study. `initializeApplicationRS === RequestStates.SUCCESS` requires both conditions to be satisfied. Please consider moving Route to `index.js`

Reply:
thank you, this is very helpful! i will update